src/FunctionApps/python/IntakePipeline/function.json

```diff
@@ -2,14 +2,11 @@
   "scriptFile": "__init__.py",
   "bindings": [
     {
-      "authLevel": "Anonymous",
-      "type": "httpTrigger",
+      "name": "blob",
+      "type": "blobTrigger",
       "direction": "in",
-      "name": "req",
-      "methods": [
-        "get",
-        "post"
-      ]
+      "path": "bronze/decrypted/ELR/{name}",
```

Comment:
was this specifically running on ELRs or is this just the last thing you were testing on?

Reply:
This was the one I was testing on when I pushed. I confirmed it worked with the other file types after pushing and have now pushed the proper `function.json` file, which uses `bronze/decrypted/{name}`.